### ISSUE TITLE
When I try to Clip a 3D Tiles with clippingPlanes,but It shows uncorrected ？the tileset's root  has transform property.

### ISSUE THREAD

**Issue by liyangis** [AnalyticalGraphicsInc/cesium#8377]
when my tileset's root has transform proterty,and X-axis, Y-axis and Z-axis of the position are inconsistent whith the   boundingSphere's center, when clip the 3D tiles,when i want to change the position through the clippingPlanes.modelMatrix,I find that there's a gap with plane and the clipped 3D tiles.But now i can't publish my 3D tileset on line,you can try it with a local data whith the origin coordinate system is far from the boundingsphere's center.I help that i have explain my question clearly,Thanks everyone.
![QQ截图20191108153036](https://user-images.githubusercontent.com/34089375/68458039-be2af100-023c-11ea-8df3-0071442ca2f6.png)

![QQ截图20191108152759](https://user-images.githubusercontent.com/34089375/68457884-6096a480-023c-11ea-98db-d72555cb3e5e.png)

**Comment by liyangis** [comment]
Sandcastle example:https://sandcastle.cesium.com/#c=7Rn5U9s89l/Rdme+ONugHJAeBLoLIaVhyrEkpduWzjeOLRKBbbmWHAg7/O/7dNiWnYTCtvvbMgPY0tO7Lz03m2jP95GLvIDGMY2mKA7ciDRgRT2gKWEhEckCCYb4jN0iMSMoIXFCOImEKyiLELuSq5dRs5mfjnzksUgkLFAHQncaUZH6xMAa5D7lwo08gm6pmGlAlnKCFarDxI0E6hNO0/BohFzPI5xLNhYsTZCk63JOBL+MNAwesgj75MpNA7GngMfshkRoF9XI4mg2OfToKT0afroftk/okA+j867XH74a3sT/uugfvcUA9MM/vJFA9Hg87XzpnF9/HbVaX6/PZx/H5+Hx4UB8CYfdE9q6O7neD0/uL4Iv1zdbX68H9GP/KP4KyE4P9m6PF7fU65zMvcOL++F1PBmG7+89STD0r4dBC/4HMx9gj8ef2scHN1vH473bkw8tnPrh7IyTw/jkYLSZdk+P+ofh2+715/T9B/JaLKb0QLw5+zE43d8c1nqX0WU0dxM0p+SWJCBjRG6NrvCFWnNqnnrtgxlcGpGk1kD/vowQ/NDoiu2zu2105QacNPQiJwHxpDmHkU89V7CkvN9sCpIkgOksYXPqE9g29LyEuIJ8ZkngjzWIU7+MHuo9zSL3CJh617CK1WvOv3S708k1UOYA8q22PzwGPmtnjErbByz15esw0n6iXo6ZT4La916hALUCx414Ppmk032WghzR9IIFaUj4IHInAfFRWSbiT8lILAKAKwBEkub7d24YB2S8iAmHHYtZA+ClSQJhMCjgymDfWt9BE7m0wk2mRHwBVlu4ZQRQkTCIBBWUKB1kkmmDEP9MAvQKdckoVWtc4YVIGSlIE1O3M/B5FUbIZ7eGsHz6AEEZVH1lBMYj0Sh2PTKYgyQGyLGNhT0XkHBpTwsPhtgbRnEKsSa9xrlKI/0QsjkJAVU9s4eSkno3xNdaAQ40XrmYg+OYcSoRSDrKS6+QY7iEqAYP9h0bSx398YcGzH4eAcbUfy48Vuqs51IUMWJMAmKsPNJbA49DiJKEutJTDek+C1iCP38YjgdYpsC9IJ65DvhGt74WC0sF+CtRR1disk8qNfPCxn03JInb15lZGpEot1d2lM6ngsMgeADPbazzE+nq+OPg/fjPg9PPJ/XME8+BVZdn6Z1ljpjG2g3T+Hc4YY5lrQvmRlvhQyVtPmLeZ5ir/ZusVfEuSGCa58IgJlU/16wypfWeZs5PZ7kxP8U+qKBqSxmv2pry6XfY08Lz9KTyPMuqHEgC4crcm2ccKCqJODNpBy/QRrFFIr/YsGyUZfCXuwbfk4Pl+PTTaPDn8enFQCs4Ew3p8qmY1hp/nwmtM1AmR0JEmkQoP+eUJFSwOG+odjNOLd4NAjtJydL0kFcnGkBTJ3oWbwFz/bFedtIkyCmW61DZ+n17ry/9UUtTZVZW1G/ljLwOi2NvgM3gyY02ZaJsIPVno41bdfVcLzB+bxTPstJ/pj50mdtFy4BX1P+/I0AFQIDKqEibSxlfa6LSy+A4oSF4ypxw7Pp+iVf1b/MgU6GlAtAmUIG/FpMVrW5XFjJ+lhjCquMZQXde7noMo1rWR7qiXsnBMqTglv4Cmr2QQkcOvfmq/JrF1sTgHcUzkijvM0jKG73yqcT1aSq9pwyF9XoupQLXCr9nLBwzx2Bv2A7zAdiVqqLCm5270ZSAe3TBM1q408go/Q1tgYfUS5hlIvmLwXHsioTebWHyI4VC6OSaYExgAdcRfsWSMA/zDHp4MDgZD8dfyilH98xoDJea8t0KUQ4dODiMGwQLlHU94HquUFcgQ7XGkaSLcrp4CfdeHAMGFy5Cvq+QQLEKFVPymgS+CQVMoawwAIEtVzOtI66tpg9UyKjEkLHQ1yh3qxqARDOWMIG6D67V2+oortvVMyM5I3Q6E1bFzOHzHOdU2GqscTqs5apSKccWDmWQaHmW5buCKLAF/Fk60tyXBHuwfQ6YRo4Uk8qrQA/+7VT5CUg0FbMeevmSLrlVfm2A05VjYAmHrlJocc1YFPmLmGuHylsVIipPG+eEVLRSp40VRxRf22gFNnU3o1BeOaCU6W2VGjtOHqsdGb/2W72xGmneqW3/vFNbgyJju8RSEExc7wZSYEwSsXB+Vqgbun1eR8K0guUb5hog3S+uEmf52EN56aGc3nLhsismjlM+cyx3qHhppVsomgKNHDNIHckt1ISiHMClnyUlL/XAwCwgOGBTs9uzqulDqf+RPYYqUf/vMHSkZkG3KvnJTHRAptBfcmej3dnErddbW6/abxtoC0pbt7X5uvWqgTZbks9egXOmi2Mps82wYOewDGnNaW92ywdiWUbzOUVesEHXS4uzuNL+lyoxnHAM9YbG2lBobGIsofksMedwnGV3jmcVhP9MZcBHKvKMshqSDRsneWKqi+DqtNSIWXkvJ1Ds2uxu228WjCony2kwTegSMX2jcwPJxpvKegiNQpiGZ/SOBCN6L0Ha0raNx2rZmsbRim/b47R6oJbKUUZeILT6cqhfKFjPKVb/VaF63Fg/qUlPqEcqmLKYWpG2f7H6/I8rz0+rzjMrjlUhKpXmaVXmwaR/aF/PmBxdS6kEXKY5nUYmbGI9YfbRZIH2wfQBWaDRggsScn1TndDwU2INZoYsOiecpYlHVIbckx8Ghr7T6bzqbmVz6FjOlNVI+Uln26+2Ou3sLM0G0PpoDePmSI18D1zhNkvXvGY+rC6eTosc0cya1GvOolovm6SA2CtRqx1uKOzRxDz9CY94Gkz0hwD7lq5Vk89wxjKu9ecY5UO68aeqleMqrOWCjDLNhJkEMAYOKNO6z7xUDUQgVgeBmo3sL0A7NQNSk5SMJm8i5t2AO+lk4uRFcBWIK68u+4aPArSRkdYCVE8BE6cTTpK5rKT2qdryIL5WxzydcC+hE6tVgSi7cIPUnqpIrRWeLK+DGRDa3d2tTPNLqW213pWXI7AaeRxZez2ykqs+B2dnPU7bhe2cqwLRZFDpCzlcPu5wJ2wOjqM/7VijMOPI5ZuTlcaed3tqd1Wb1X2DoYHptvCbTrdeEb0imW4as9ixE8xzveeR2chqN5qXfGjFLNIopzwSeMK4RuL9BUGW282n8K/6pbx9NVzYfWoeHZUCbh/Kn3s533mLb4Isp1cp6gmRc9c96BTzDm7NfE1D5to10Cs/odnTWrtlLYZ49nxd1aQXjRc7HLRH3mnQf9AwZomQ/ZoDaRnqD1ACf21OUmiWBPY4z1j4a5YyLXPLEj5NpIm3UTKduM5WB9p08wsObkdhLEc40XQbbcV31vKEJT5JNvQl2N58qJClcmhearugcaDQVW64ARTWbegkfT8gyxQ3BIu3UadENduaMCFYaO9WyarWnJSkvmKR2LhVAxA5NQisrxc7zUy7Oz6dI+rvXr6ofKS+fAE+5nIOO1dpoHreyxfvdpoAXzom4x9YPAUxA3chQWbtdx/1IsZ4pwmvy6cM2wCuedrRnwwQtFTuhqyKAMNi6bKgbPvjb0PHxPaKD76KPY0nw6qNIWBTCjgj3s2E3YFkCoeUTDZr8G6TVWAEPOWRXGS40C3gNqopOjVgAMlkUszz5urUE5mRbeE6XpbTyXoWNIA6grg+Iy2uTXAZ/Qc

Browser:chrome  78.0.3904.87

Operating System: WIN10
![QQ截图20191108142139](https://user-images.githubusercontent.com/34089375/68454318-31c80080-0233-11ea-8d0b-025d7fb937b1.png)
![QQ截图20191108142155](https://user-images.githubusercontent.com/34089375/68454340-43110d00-0233-11ea-8df2-31a7a365d138.png)

**Comment by liyangis** [comment]
When I move my mouse ,but clipped plane is from left to right,but not from top to down,why?

**Comment by liyangis** [comment]
This one is looks like correct,but the origin position 's axis of two is consistent with the bounding center,only one axis is not consistentent.
https://sandcastle.cesium.com/#c=7VrrctvGFX6VrTMTkgkFXiQ6Diml1YWW6bEuFWm5dpTxgMCKXAnAIrsLSXRHL9CZdvoW7a++QCdv00xeo2cvABcgIFOJ/aMzzUxkEjh79ly+c/bbXbZaaNf3kYu8gMQxiWYoDtwIN+GJ+oBmmIZYsAUSFPE5vUVijhHDMcMcR8IVhEaIXsqnF1GrlY2OfOTRSDAaqAGhO4uISHxsZI1yn3DhRh5Gt0TMtSBNOHaUqkPmRgLtY06S8OUYuZ6HOZdmLGjCkJzX5RwLfhFpGWdEI8fHl24SiF0lPKHXOEI7qIYXL+fTQ4+ckJej1x9GnWMy4qPorOftj56OruM/ne+//NYBoR/9w2spRI4ms+7b7tnVu3G7/e7qbP5qchYeHQ7F23DUOybtu+OrvfD4w3nw9up6693VkLzafxm/A2UnB7u3R4tb4nWPb7zD8w+jq3g6Cp9/8OSEoX81CtrwbzD3QfZo8rpzdHC9dTTZvT1+0XYSP5yfcnwYHx+MN5Peycv9w/Db3tWb5PkL/I1YzMiBeHb64/Bkb3NUG1xEF9GNy9ANwbeYgY8RvjWxcs7Vs3rNU1/3IQ0uiTCrNdGfLyIE/5Hoku7Ruz66dAOOm/ohxwH2ZDpHkU88V1CWf99qCcwYaDpl9Ib4GF6b+TyGXYHfUBb4Ey1Sb1xE9w1lZKulTXTwncCRXzdj9EP9ZfNgQgIMCeExWEDZEbkjkRwtHeQeBqDsGEcd9TXzXoL2ZHoFoziIfF/bGx2Bl7VTSiRyApr48isoVihTX46oj4PaD4Nl+NQTGG6C4+NpMtujCUQhmp3TIAkxH0buNMA+ykcE+zM8FosA5JYCgiXZ+zs3jAM8WcSYwxvLWCPgJYxBEQ2Xcnmx79s/QBwzb4XLZli8BVPbTts4oOpoGAkiCFYxSD3T6cT+qRQYLMMla1w94yY9aKwkTUXezqFiVBEin96aieWnF1DSQRFpY0g9jsax6+HhDXhihOp2shzPBSVc5tPS40DljqI4gUqVmKtfJpH+ENIbHIKqRpoP5SXxrrGvowIWaL3yYSbuxJQTqUDOozB+iVKsQU8A/Pt1W0sDffmlFkz/e0DYIf5j5R0VzkbmxbLCTErAjdIhgwp5J4QaY8SVSDVT79OAMufNi9Fk6MgGuhvEc7cO2Og1KrXQRABesRpaqskeqcLMlzned0PM3H3d12USsYK9yqMEnyoOo+AekNuswomEuvNq+Hzy/uDkzXHaKNAZmOrydHGgKRCTWMMwiT8FCDMtlRDMklaCoVw0H0jvI9LV+UTZKqALGpi2eZkQ0+gfm1bZ0gbrpfP1aZbM17EPISjmUtarzqb89CnyaelZv6k8LrOqB+JAuLL3Zh0HFhUmTk3bcRZoY/kKFrrlCytHaQf/esfoW7tYjk5ej4fvj07OhzrAqWtIL77KaB3x56nTugOlfjAsEhahbFw956GSdTI6tpNaatluFNhNSi5N99nqJNdwLAaWbQF1/Yl+XE9YkM2YX4fy2d+33+1LPGpvisbKFfX7fEeu0lK3X0DO4JMbbcpG2UTqz0bHaTfU50Yzr/LX66xU+UiN0jajsv2prCxTudT4g6VdUpw3xAdy3l9yJaeE+PxeOgxCoMpgQ+NUoV5DoEDinJiREErkBnPH9f2cqTYtBOxYuQcYwSzw1zKyAKd+4UFqz4pBjqJ6Y9jU5OmeMVT7+gAdHOQqK1UK9egvgCOHBDYysKUpW1jSpjI1esfxHDNVdkZJ/sUgP4q5Pklk2eSlHP0881KJ64B/oDSc0LrR3rTx8gLMlaEiwpufudEMA4Z7UBJtp9tMZ/oKbQFCGgUzfkxcWbpJEFgv1BsBOzd+SVm4D+0LM+u17Lu/MzMfuYKRuy0Hg6KA17P4USqcTEPWFVPp0cHweDKavM13aL1BQRPYQeY3sohw2O4AzNwgWKCUJAJgXaH2m2bWGkdy3qXlzoru3TgGDS7sOn1fKYG1PVRGyT2ppzxVKgsGQB+UT9NcIa5zrQcUpilGbrnmp+5DU55ImUDtvCuDVl73uRSmyZpjMpsLi11k8tl6UC+Y1azAqaOdKs6SL0cnlHWl/Vn17xIKx3bwY21WW1/uWGzFb5J6wJ1Lcof95wxIz4Tm0A+LTUVAi+o19FdVzwvq/phIFhhJR0oj1ZRG2srv7eIFnaguPSFyyzeAf7aLsQxwNBPzAfr6a7JSD9n2EEYXhgGK6qQsZsvt5GLZrrHZXqo2XZhErcemqqDzlntZMkTZ1Ucl2tQenACN4qBSdvMyCHTrujV91UFWl+o0muX6MjLe/xgZ71WpSC3OWRMEU9e7hmYfYyYW9Y9xsabeIVVNYdh+/hChQkhvCcrcWR12X6KKMpIe3/UlnFcl8k+s5TwXkfTowYkTPq9b8CmgetlIf/nHv3/+yz9//vvf/vPTX3/56V+rExdw+F501oeiDcd+sZtWRBR6+5TeVYMR3q+Fyc16d0u2JP230yvhaXmNVah8O3z16uRNDpabjUpN9xWT4ABqnlPiVzqmyAvUDqnyR3GA9E+VM7YnpY6UD6sy++PAvC82rnLU/FrQrNvC/g+dzwmds+HBZ8VNnrSn+9mGQ4GrsVug7kvWjhmjLLe6epBJGmAnoDPzdmBteu5z+3O5B1Y7if+VHfBn3AiqGk3JQhnhlOzvAM8Yxhw2vt1Np/3N1tbTzrdNtAWV0WtvftN+2kSbbWnnYKnT0K4cm5w7gp7BY1gB6p3NXn5ALAlado6e7asg1isPNYus2jBJymhmb2qtTaXGnsxC5mNJYxqsHE+UOvGaFC0ClruyX7b4WjbB8q1tbt/+ZskoCr/a9hJGVibTJ45uIM14Vngews4sTMJTIOTBmHyQIh2Z2+ZD+4eK/b3FM2zE6fAAEZBH7Rmx1eHLpH4D0X4Myf5VBPvhZH2ES6/Bo1UxpTVV0lvXp86l7Psz0+aPUuZH0mWLqRYY73ps9960f1hcT6m8mJVeCeTDtLPIlE2s7099NF2gPUh9gBdovOACh1yfpE5J+JpZFwcjGp1hThPmYdUhd+W198ivd7tPe1vpPWks7zzVledaYztPt7qddCxJL0j10JrjtMbqSvLAFW4rdxrXyi5Tl59Olj2ile6frziNaoP0pB/cLlWt3nAzwy5h5tN7+OjMgqm+5rZPkXVosjuGiaxr/WMDhSF90kIUf+OqrOUDWWXaCHNSTSkAULZ1n3qJOrCHWh0G6ux+bwHRqRmRmpzJRPI6ot41wEk3k3q2CJaJuPKsaM/YsRRtplNrB4qjwIiTKcfsRq6k9qja6kVxreHwZMo9RqYWVYEqO3eDxD71l1FbIlmev6VCaGdnp3DbnGtt5XFXKAeGyPHDyjrVynJQfYzObrVOG8J2z1WFaDqoxEIml51Ku1N6A8DRP1ywrmoMkPOnVVYbe9yJleT0QLN6zxwgML2286zbaxRcL3imSWNaO3aDeSx6HjjCLofRTQ5DJXdlJjj5M9g1TtWl3t/gyCrdXMd+xZcy+mqssHlqVh2FBdwelH0eZHZnFN8UWTZfYVFnWN4L7gJTzBhcxTWIlsyia6RLf+Jh3ybalHV512Lf/6o16UnzyTaH6OHvtOgfSBhTJiRfq0NbhvUHZgK8tqYJkCXheJynJnyRtkwr3XIJnzGZ4j5is6lb34LNXvo/ANyuwliemUezPtqK76zHU8p8zDb0+Z398r4wLZGXujnaBcSBAKvccANYWPvAJH0/wKszbgga91E3N2v6akqFoKH9tjitouY45/UljcTGrTp0lkcFgXW7vt1Ko7vtkxtE/J2LJ4WfYF08AYy5nMObyyRQnPfiyXfbLZDPDZP1DyaegJuBu5Ai8853r/RDx3G2W/B1dZQxG8S1Tdv6ShsBpXI35KoIMjSWkIVg2z9Oauqa6Jf8IEmZp/WkWnUyBLyUDs6xdz2ld+CZ0iE9k2QNvtvTKjEMSHmgFxkrNAXso5qapwYGINlMlhcoN2rUmsZIWlhly2o7qTZBC6ghiOsxMuM6BRfRfwE

**Comment by liyangis** [comment]
hi,look at this one,is appear
![QQ截图20191108172956](https://user-images.githubusercontent.com/34089375/68465425-6f398780-024d-11ea-9e96-acfaa91fcdf5.png)

https://sandcastle.cesium.com/#c=7Vpbb9zGFf4rUxfIcuMV9yKt7exKaXVZy2tYl0pruXYUGFxydnckksMMh5LWgYCi6EuLNE0f2oc2RZ+KFkUQ5DFNm3/jy2P/Qs/MkNwhl5RXif0QoAEiczlnzpwz5ztnvplhvY7WHQdZyHZJEBB/jALX8nEN3sgHNMbUw5xNEaconNBzxCcYMRwwHGKfW5xQH9GReHvs1+tpb99BNvU5o67s4Fljn/DIwbFsrNwhIbd8G6NzwidKkEYhNqWqbWb5HG3ikETe/UNk2TYOQ2HGlEYMiXGtMMQ8PPaVjNmnvungkRW5fF0KD+gp9tEaquDp/clw2yZ75H7/4bN+c5f0w75/0LY3+7f6p8HPjzbvv2eC0EfO9qkQIjuDcetx6+DkyWGj8eTkYPJgcODtbPf4Y6/f3iWNi92TDW/32ZH7+OR05clJjzzYvB88AWV7W+vnO9NzYrd2z+zto2f9k2DY9+4+s8WAnnPSdxvwrztxQHZn8LC5s3W6sjNYP9+91zAjx5vsh3g72N06XI7ae/c3t7332iePorv38G0+HZMtfmf/o97exnK/0j32j/0zi6Ezgs8xAx99fB7PlXkk3xkVW/7chDBYxMesUkMfH/sI/iP+iG7Qiw4aWW6Ia+pliF1si3D2fYfYFqcs216vc8wYaNpn9Iw4GJrj8WyGLY4fUeY6AyViVI/9y6o0sl5XJpr4gmPfMeI+6qX6sbw1IC6GgIQBWEDZDrkgvugtHAxtDEBZix015c/UewHaveEJ9ApB5IPKRn8HvKzsUyKQ49LIET9BsUSZ/LFDHexWPuzOpk++ge7x5Dh4GI03aASz4I+PqBt5OOz51tDFDsrOCHbG+JBPXZCbCXAWpe0Xlhe4eDANcAgtmrGxgB0xBknUm8llxT5ofAjzmHrLLTbG/DGY2jAbsQMyj3o+J5xgOQeJZyqc2NkXAt3ZdIkcl+/CODzoUErGGXk+gYyRSYgceh4PLJ7uQUq7eaQdQuixfxhYNu6dgSexkKEHy7QtUBKKeGp6TMjcvh9EkKkCc8Yo8tWDR8+wB6qqSTykl8Q+xY6aFbBA6RUvU3EzoCERCsQ4EuMjlGANagLg3zF0LVX0zjtKMPnvCmGTONeVN+V0VlMvZhkWhwTcKOzSLZE3PcgxRiyB1HjoTepSZj661x/0TFFA191gYhmAjXa1VAuNOOAVy66FmvSecprDWYw3LQ8za1PVdRFELGEv4yjAJ5MjVnAJyK2V4URA3XzQuzt4urX3aDcpFOgATLXCZHGgCRCjQMEwCt4ECFMtpRBMg1aAocxsXhHea4Sr+YailUMXFDBl8ywgcaG/blhFSesuFs6H+2kwHwYOTEE+liJfVTTF05uIp6Zn8aJyvcjKGohdbonam1YcWFQY34/LjjlFS7MmWOhmDVqMkgp+cy3Wt3Cy7Ow9POw93dk76qkJTlxDavGVRqsZv5s4rSpQ4gfDPGI+SvsZGQ+lrJnSsbXEUs32WIFepMTSdJmuTmINx7yr2eZSyxmo10bE3HTE7DqUjf6m3rYp8Ki8yRsrVtQPshW5TIuhN0DM4Mnyl0WhrCH5Z6lpNqryuVrLqvzuOktVXlOjsC1W2XhTVhapnGn8UNMuKM4j4gA578y4kllAfH4iHAYhUBVjQ+FUol5BIEfizIARD1LkDIem5TgZU3VaCNjRYg8wglHgr2ZkDk6d3IvEnjmDTEn1DmFTk6V7saHK1yvoYDeTWYlSyEdnChzZI7CRgS1N0cKSFJVhrPcwmGAm0y5Wkm3oZnsxyyGRSJuslKnep15KcTXhzyj1BtSItdd0vNwDc8VUEW5PDix/jAHDbUiJhtmqJSO9i1YAIdWcGR9FlkjdyHW1BtnCYecWjijzNqF8YaY1i7r7o3jkHYszcrFiYlDkhkY6f5RyM9WQVsVEur/V2x30B4+zFVptUNAAdpDZjSwiIWx3AGaW605RQhIBsBaX+8141EqIxLgzy8053etBABos2HU6jlQCa7snjRJ7Ult6KlXmDIA6KN4msUKhirXqkBsmN3GzJT/xHmryQMi4cuNdOmfFaZ+JYBKrCSbjCdfIRSqfLgdGzqxaCUxN5VN+lHo9m4+mJxJLeTTv4QgyR3fxdXVW2T/nmnKOjkaq8BQamu8hw/zy81+//OO/Xnz2+1d/+tV///PJqy++eP71L55//c8Xv/vlyz989fzrb179/d8vfvOPV3/+EgRefP5VLPPNJ7rMi0//+vIvfwOB59/+9tW3X84PlGaJ8rJw+sNoCCI2N5QXNTQXhgXDPF8jrxcCzdBCBAUaUAeJjaE5IhfYucuAXA5opsrAol6C3Lx6VWLmVU9y6n4WCbbtC2MLI10TRurKL/UiCTqRITwhYmvdhX9W8/PlYn/MJ1108yaZqzvpNhx657pBuhqkaM5m2/bpbFnE8TZeLocfz8cwqV6wwhV7WdBF2tVBBdrkWQcBuhqCSrFqFqGpZagl4N0m0laDZrVWrC/d9HRet+lpl6lILM5Y47pDyz6FRTXAjE+N13HemtqJlg0R76qyhzUlQmrrVeTOfLfLAlWUkeSYtCPgPC+RfaPRpsyMJEc8ZhCFE0ODTw7Vs0qm6tSLzz69qgZpip7y5uJQ1OHYmStMxfJQXIf0ohyM0L4QJpeN1oqo/Opvs13Ah7May1D5uPfgwd6jDCyXq6WaLksGwS7kfEiJU+qYJImQO6TMH8m1kj9lzuieFDpS3K3M7NcD87JoHZlHzXcFzaIl7P/QeZvQOehtvVXcZDdHyblB1aTAidk5bJFmuyPMGGWZ1dWGSFIXmy4dx61dbXN5mTkHEWcNcsf2QzlpeIsbbpmjCVkoopaC4W3hMcM4NJaarWWzcXtl5VbzvRpagcxoN5ZvN27V0HJD2Nmd6YxpV4YxTkxOD+A1rABGc7md7RAIgpbeV6T7V5jruZeKRZZtTAVljEevKa01qUYfTEPmdUljMlkZnih04gUpmg8sd+5cQuNr6QCzVt3cjv5Lk5E0fb7sRYzMDaZOdi1XmHEn996DHbAXeftAyN1D8kyINEVsc2ILnaNoPENHnJoesV85xU5KbNX0pVLfg2hfh2R/J4J9dbBew6UX4NEymZKcKqiti1PnQvb9lmnzaynzNemyxlRzjHcxtnsZl39YXPepuAAXXnHkwLBjP06bQN1TO2g4RRsQehdP0eE05NgL1Yn1kHgPmXZB06f+AQ5pxGwsK+S6+Lyg7xit1q32SnIfHYi7ZXm1vFDf5q2VVjPpS5KLaNW1Ypr1Q3n1u2Vxq5459aynl9azp71Zjagn++eTkPqVbnKjAm4XqpYtYTzCOmHx01N4NMfuUH1OoJ/Wq6lJ73IGIq/VRx0SQ+pEi0j+Fsq0Fi9Elikj4hsBSgGAoqw71I7kxQjkas+VdyQbU5idSixSESPFM3nqU/sU4KSKiZEugkUiljiT24jtmInWkqGVA/leYMTeMMTsTKykeq/K/IV8pSrOYUKbkaFGVSDLjiw30m9XxKzNkCzOORMhtLa2lrvVz5S24nmXKAeGGOKrlTXLlWWgeh2drXKdOoT1misTMa6gAgupXHr6bw3pGQBHfSCiXYnFQM6eSGll7HoHg4LTA81q3zGBwLQb5p1Wu5pzPeeZIo1J7ugF5rroueKqoBhGZxkMFdxJxpOTPete4PZC6P0ejszTzUXsl3wppa+xFTpPTbMjt4DrndLnbmp3SvHjJEvHyy3qDIv713VgiimDK7luUpLp7MbShZ/S6Le2OmWd3Wnp9+xyTbpRu7Eawuzh95XoT4kXUMYFXzOgLMP6AyMBXuvDCMgSN+0wTEz4cVIytXCLJXzMRIg7iI2HlrECm73kfwC4noWBuJvwxx20Elxor4eUOZgtqfM7vfEyNywRl+cZ2gXEgQCrXLJcWFg7wCQdx8XzIy5xGnRQKzNq0jSknFNPb80PK6k5zng9oj5fOpdn++KowNW+YlitJ7O76pAzRJy14xu5T92ObwDGrDCEllHkSs57fOP91TrIZ7qJ/AcT98BN15oKkUnz/QfqpWmaq3X4Od8rNhvElU2r6tMBBJTKWhKrIsjQQEAWJlv/CKymcqJT8OGXNE/pSbSqYHBoFA5OsH06pBfgmdQhPBNkDX7rw0oxDEi5ohbFVigK2EEVOU4FDECimMwuqs5krwWNEbSwzJb5clJughKQXVCo+oiIqxAc+/8D

**Comment by OmarShehata** [comment]
Thank you for the Sandcastle examples @liyangis ! It helped narrow down the problem. I've confirmed this is a duplicate of https://github.com/AnalyticalGraphicsInc/cesium/issues/8053, and I've opened a fix here https://github.com/AnalyticalGraphicsInc/cesium/pull/8392.

Your initial issue with the clipping happening on the wrong axes was just because the visual plane and the actual clipping plane were not using the same axis. I have a code example in the PR linked above showing how to clip along all 3 axes.
